packages/xo-web/src/xo-app/backup/overview/tab-jobs.js

```diff
@@ -99,7 +107,13 @@ const _runBackupJob = ({ id, name, nVms, schedule, type }) =>
         })}
       </span>
     ),
-  }).then(() => (type === 'backup' ? runBackupNgJob({ id, schedule }) : runMetadataBackupJob({ id, schedule })))
+  }).then(() =>
+    type === 'backup'
+      ? runBackupNgJob({ id, schedule })
+      : isMirrorBackup({ type })
```

Comment:
`const method = isMirrorBackup({ type }) ? runMirrorBackupJob : runMetadataBackupJob`

```suggestion
      : method({ id, schedule })
```